components/analytics/true_up_review.tsx

```diff
@@ -0,0 +1,228 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useEffect} from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import {useDispatch, useSelector} from 'react-redux';
+
+import moment from 'moment';
+
+import classNames from 'classnames';
+
+import {submitTrueUpReview, getTrueUpReviewStatus} from 'actions/hosted_customer';
+import {
+    isCurrentLicenseCloud,
+} from 'mattermost-redux/selectors/entities/cloud';
+
+import {
+    getSelfHostedErrors,
+    getTrueUpReviewProfile as trueUpReviewProfileSelector,
+    getTrueUpReviewStatus as trueUpReviewStatusSelector,
+} from 'mattermost-redux/selectors/entities/hosted_customer';
+
+import useCanSelfHostedSignup from 'components/common/hooks/useCanSelfHostedSignup';
+import CheckMarkSvg from 'components/widgets/icons/check_mark_icon';
+
+import './true_up_review.scss';
+import {GlobalState} from '@mattermost/types/store';
+import WarningIcon from 'components/widgets/icons/fa_warning_icon';
+import {isCurrentUserSystemAdmin} from 'mattermost-redux/selectors/entities/users';
+import {getLicense} from 'mattermost-redux/selectors/entities/general';
+import {TrueUpReviewProfileReducer, TrueUpReviewStatusReducer} from '@mattermost/types/hosted_customer';
+
+const TrueUpReview: React.FC = () => {
+    const dispatch = useDispatch();
+    const isCloud = useSelector(isCurrentLicenseCloud);
+    const isAirGapped = !useCanSelfHostedSignup();
+    const reviewProfile = useSelector(trueUpReviewProfileSelector);
+    const reviewStatus = useSelector(trueUpReviewStatusSelector);
+    const isSystemAdmin = useSelector(isCurrentUserSystemAdmin);
+    const license = useSelector(getLicense);
+    const trueUpReviewError = useSelector((state: GlobalState) => {
+        const errors = getSelfHostedErrors(state);
+        return Boolean(errors.trueUpReview);
+    });
+    const {getRequestState: statusRequestState}: TrueUpReviewStatusReducer = useSelector((state: GlobalState) => state.entities.hostedCustomer.trueUpReviewStatus || {complete: false, due_date: 0, getRequestState: 'IDLE'});
+    const {getRequestState: profileRequestState}: TrueUpReviewProfileReducer = useSelector((state: GlobalState) => state.entities.hostedCustomer.trueUpReviewProfile || {content: '', getRequestState: 'IDLE'});
```

Comment:
Theses seem like duplicates of `reviewProfile` and `reviewStatus` on lines 38 & 39. I don't think we need the default values; those pieces of state are guaranteed to be defaulted already by your `initial_state.ts` changes. They might not be defaulted in some tests, which might have caused errors. If that's the case, I'd recommend setting the default state in the tests rather than adding a default here.

Reply:
Ah, yes I suppose they would be duplicates! I forgot the state for the request is just set in the profile/status entities. I'll remove this and use the existing one.